src/js/models/home.js

```diff
@@ -76,5 +97,15 @@ export default class Home{
 			alert("seniorGetOpps: " + error);
 		}
 	}
+
+	async getJuniorInfo(juniorUid){
+		try{			
+			await db.collection('Profiles').doc(juniorUid).get().then(doc => {
+				this.juniorInfo = doc;
+			});
+		} catch (error) {
+			alert("seniorGetOpps: " + error);

```

Comment:
Think you copied but forgot to change the message

Reply:
Done